src/app/api/gql/fragments/events/fragment-set-polling-source.graphql

```diff
@@ -0,0 +1,123 @@
+fragment SetPollingSourceEvent on SetPollingSource {
+  fetch {
+    ... on FetchStepUrl {
+      url
+      eventTime {
+        ... on EventTimeSourceFromPath {
+          pattern
+          timestampFormat
+        }
+        ... on EventTimeSourceFromMetadata {
+          __typename
+        }
+      }
+    }
+    ... on FetchStepFilesGlob {
+      path
+      eventTime {
+        ... on EventTimeSourceFromPath {
+          pattern
+          timestampFormat
+        }
+        ... on EventTimeSourceFromMetadata {
+          __typename
+        }
+      }
+      cache {
+        __typename
+      }
+      order
+    }
+    ... on FetchStepContainer {
+      image
+      command
+      args
+      env {
+        name
+        value
+      }
+    }
+  }
+  read {
+    ... on ReadStepCsv {
+      schema
```

Comment:
It looks `schema` is in every branch, can we move it 1 level up?

Reply:
No, we can't move this field, because it's a union.